src/makeRobot.js

```diff
@@ -38,7 +38,67 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return (
+        `name: ${this.name}, chip version: ${this.version}, wheels: ${wheels}`
+      );
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(move = 1) {
+      if (move >= 0) {
+        this.coords.y += move;
+      }
+
+      return this;
+    },
+
+    goBack(move = 1) {
+      if (move >= 0) {
+        this.coords.y -= move;
+      }
+
+      return this;
+    },
+
+    goRight(move = 1) {
+      if (move >= 0) {
+        this.coords.x += move;
+      }
+
+      return this;
+    },
+
+    goLeft(move = 1) {
+      if (move >= 0) {
+        this.coords.x -= move;
+      }
+
+      return this;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
```

Comment:
```suggestion
      this.coords.x = SERVICE_COORD_X;
      this.coords.y = SERVICE_COORD_Y;
```
Remember to use constants instead of magic numbers. Place them at the top of your function.